src/validate.ts

```diff
@@ -48,14 +60,18 @@ function outputErrors(
         "action parameters as `false` in the workflow file.\n"
     );
   } else {
-    core.startGroup(`❌ Commit (${sha})`);
+    const subject = (message.match(/^.*$/m) ?? [""])[0];
```

Comment:
duplication of logic bump.ts:153

Reply:
See: https://github.com/tomtom-international/commisery-action/pull/211#discussion_r1105441238

Reply:
I've reworked the other two, but this one will need to stay, as there isn't a `ConventionalCommitMessage` object available here.